src/neo-vm/OpCode.cs

```diff
@@ -310,74 +310,270 @@ public enum OpCode : byte
 
         #endregion
 
-        #region Old opcodes
+        #region Slot
 
         /// <summary>
-        /// Puts the input onto the top of the alt stack. Removes it from the main stack.
+        /// Initialize the static field list for the current execution context.
         /// </summary>
-        TOALTSTACK = 0x6B,
+        [OperandSize(Size = 1)]
+        INITSSLOT = 0x56,
         /// <summary>
-        /// Puts the input onto the top of the main stack. Removes it from the alt stack.
+        /// Initialize the argument slot and the local variable list for the current execution context.
         /// </summary>
-        FROMALTSTACK = 0x6C,
+        [OperandSize(Size = 2)]
+        INITSLOT = 0x57,
         /// <summary>
-        /// Duplicates the item on top of alt stack and put it on top of main stack.
+        /// Loads the static field at index 0 onto the evaluation stack.
         /// </summary>
-        DUPFROMALTSTACK = 0x6D,
+        LDSFLD0 = 0x58,
         /// <summary>
-        /// Copies the bottom of alt stack and put it on top of main stack.
-        /// </summary> 
-        DUPFROMALTSTACKBOTTOM = 0x6E,
+        /// Loads the static field at index 1 onto the evaluation stack.
+        /// </summary>
+        LDSFLD1 = 0x59,
         /// <summary>
-        /// Returns true if the input is null. Returns false otherwise.
+        /// Loads the static field at index 2 onto the evaluation stack.
+        /// </summary>
+        LDSFLD2 = 0x5A,
+        /// <summary>
+        /// Loads the static field at index 3 onto the evaluation stack.
+        /// </summary>
+        LDSFLD3 = 0x5B,
+        /// <summary>
+        /// Loads the static field at index 4 onto the evaluation stack.
+        /// </summary>
+        LDSFLD4 = 0x5C,
+        /// <summary>
+        /// Loads the static field at index 5 onto the evaluation stack.
+        /// </summary>
+        LDSFLD5 = 0x5D,
+        /// <summary>
+        /// Loads the static field at index 6 onto the evaluation stack.
+        /// </summary>
+        LDSFLD6 = 0x5E,
+        /// <summary>
+        /// Loads the static field at a specified index onto the evaluation stack. The index is represented as a 1-byte unsigned integer.
+        /// </summary>
+        [OperandSize(Size = 1)]
```

Comment:
Any reason for OperandSize be present only in a few opcodes?

Reply:
Yes, because the others have a defined value